src/Version.h

```diff
@@ -12,7 +12,7 @@
  * Whenever a number that precedes <revision> is incremented, all subsequent
  * numbers should be reset to 0.
  */
-#define CPPTRAJ_INTERNAL_VERSION "V6.4.1"
+#define CPPTRAJ_INTERNAL_VERSION "V6.4.2"
```

Comment:
Since you didnt actually change any cpptraj code you don't need to increment the version. Can leave at 6.4.1.

Reply:
I just merged in a PR where I also incremented to 6.4.2 so this should be fine.